examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/MCEndpoint.h

```diff
@@ -41,6 +41,9 @@
 - (BOOL)hasCluster:(MCEndpointClusterType)type;
 - (MCCluster * _Nullable)clusterForType:(MCEndpointClusterType)type;
 
+- (NSString * _Nonnull)description;
+- (void)logDetail;
```

Comment:
Unlike description, logDetail is a non-standard function. Can't we just log the endpoint description by calling ChipLogDetail/Log directly, when we need to? It is one line either way. I'd rather not expose public APIs if they are not really needed.

Reply:
We make logDetail available in a number of functions across iOS, Linux and Android versions of Endpoint and other classes. For consistency I added it here as well.  Also [endpoint logDetail]; is more compact than ChipLogDetail(AppServer, "MCEndpoint Details: %@", [self description]);